### PR TITLE
Kp/regressor

### PR DESCRIPTION
**Pull request by kprokofi**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -9,7 +9,7 @@ tensorboard
 onnx
 glog
 torchvision
-torch
+torch==1.7.1
```
Comment (sovrasov) [inline requirements.txt]: Why do you need exactly this version?

Comment (kprokofi) [reply requirements.txt]: I ran into a problem on the cluster when leaving the torch w/o version. So, I hardcoded a version of what I have locally.

Comment (kprokofi) [reply requirements.txt]: It is also a good reason to highlight the version explicitly to not face the problems with different versions of the PyTorch

Comment (kprokofi) [reply requirements.txt]: If it's better to work with 1.8 PyTorch, I will try to migrate to 1.8 in future PR

File: torchdet3d/utils/utils.py [hunk 2]
```diff
@@ -115,25 +109,42 @@ def load_checkpoint(fpath):
...
+            key = key[len(prefix):]
+        return key
+
     if file_path:
         check_isfile(file_path)
     checkpoint = (load_checkpoint(file_path)
                        if not pretrained_dict
                        else pretrained_dict)
 
+    if 'classes_map' in checkpoint:
```
Comment (sovrasov) [inline torchdet3d/utils/utils.py]: Where do you use it?

File: requirements.txt (hunk 1 above)
Comment (sovrasov) [reply requirements.txt]: I've launched training on cluster with torch==1.8.1, will see how it works

File: torchdet3d/utils/utils.py (hunk 2 above)
Comment (kprokofi) [reply torchdet3d/utils/utils.py]: sorry, I've decided to update this function from deep-person-reid with the latest functionality of printing from LB, but have forgottenten to delete this

**Pull request merged**